command/server.go

```diff
@@ -1682,11 +1682,17 @@ func (c *ServerCommand) Run(args []string) int {
 				level, err := loghelper.ParseLogLevel(config.LogLevel)
 				if err != nil {
 					c.logger.Error("unknown log level found on reload", "level", config.LogLevel)
-					goto RUNRELOADFUNCS
+				} else {
```

Comment:
Arguably this should be fixed in its own PR, thoughts?  The issue as I see it is that `goto RUNRELOADFUNCS` only makes sense when we're unable to re-read the config, because the reload steps we take after RUNRELOADFUNCS aren't exclusively dependent on the config file itself having changed - they rely on TLS certs changing or license file changes.  But we have some cases where folks have copied the `goto` pattern thinking it's what should be done whenever any error occurs as part of reloading.  This is undesirable: just because the user gave a bogus loglevel or whatnot, that shouldn't impede reloading other things like Census or the canary write interval.